example/src/NativeMRecExample.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Platform, StyleSheet } from 'react-native';
+import AppLovinMAX from '../../src/index';
+import AppButton from './components/AppButton';
+
+const NativeMRecExample = (props) => {
+  const { adUnitId } = props;
+  const { isInitialized } = props;
+  const { log } = props;
+  const { isNativeUIMRecShowing } = props;
+  const { isProgrammaticMRecShowing } = props;
+  const { setIsNativeUIMRecShowing } = props;
+
+  return (
+    <>
+      <AppButton
+        title={isNativeUIMRecShowing ? 'Hide Native UI MREC' : 'Show Native UI MREC'}
+        enabled={isInitialized && !isProgrammaticMRecShowing}
+        onPress={() => {
+          setIsNativeUIMRecShowing(!isNativeUIMRecShowing);
+        }}
+      />
+      {
+        isNativeUIMRecShowing &&
+          <AppLovinMAX.AdView
+            adUnitId={adUnitId}
+            adFormat={AppLovinMAX.AdFormat.MREC}
+            style={styles.mrec}
+            onAdLoaded={(adInfo) => {
+              log('MRec ad loaded from ' + adInfo.networkName);
```

Comment:
We usually reference MRECs when in text as "MREC"
```suggestion
              log('MREC ad loaded from ' + adInfo.networkName);
```